### PR TITLE
Add `ky/umd` TypeScript definition

### PR DESCRIPTION
**Pull request by rifler**

### CODE REVIEW

File: umd.d.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+declare module 'ky/umd' {
+	import ky from 'ky';
+
+	export default ky;
```
Comment (sindresorhus) [inline umd.d.ts]: You can just export it directly:

```js
export {default} from 'ky';
```

File: umd.d.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+	export * from 'ky';
```
Comment (sindresorhus) [inline umd.d.ts]: I'm not that familiar with export syntax, but are you sure this doesn't also export the default?

File: umd.d.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+declare module 'ky/umd' {
```
Comment (sindresorhus) [inline umd.d.ts]: I don't think this is needed.

Comment (rifler) [reply umd.d.ts]: you are right

File: umd.d.ts (hunk 2 above)
Comment (rifler) [reply umd.d.ts]: Only `export * from 'ky';` doesn't work for my `typescript@3.1.6`

**Pull request merged**